<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: #f0f0f0;
        font-family: 'Segoe UI', Arial, sans-serif;
    }
    QLabel {
        font-size: 16px;
        color: #333333;
    }
    QComboBox, QCheckBox {
        font-size: 14px;
        padding: 5px;
    }
    QPushButton {
        background-color: #70B9FE;
        color: #ffffff;
        border: none;
        border-radius: 5px;
        padding: 8px 15px;
        font-size: 14px;
    }
    QPushButton:hover {
        background-color: #5aa9fe;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="settingsLabel">
     <property name="styleSheet">
      <string notr="true">
       font-size: 20px;
       font-weight: bold;
       color: #333333;
       padding: 10px;
      </string>
     </property>
     <property name="text">
      <string>Application Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="languageLabel">
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="languageComboBox">
       <item>
        <property name="text">
         <string>English</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spanish</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>French</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="fontSizeLabel">
       <property name="text">
        <string>Font Size</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="fontSizeComboBox">
       <item>
        <property name="text">
         <string>Small (12px)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Medium (16px)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Large (20px)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="autoSaveLabel">
       <property name="text">
        <string>Auto-Save Interval</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="autoSaveComboBox">
       <item>
        <property name="text">
         <string>1 Minute</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 Minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Off</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save Settings</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
